lib/WebpackOptionsApply.js

```diff
@@ -264,6 +264,7 @@ class WebpackOptionsApply extends OptionsApply {
 				const cheap = options.devtool.includes("cheap");
 				const moduleMaps = options.devtool.includes("module");
 				const noSources = options.devtool.includes("nosources");
+				const debugIds = options.devtool.includes("debug-ids");
```

Comment:
```suggestion
				const debugIds = options.devtool.includes("debugids");
```

Should we make this `debugids` instead? I am thinking it is probably better because `-` is sort of like a delimiter with the `devtool` option.